### ISSUE TITLE
Parser skips tables containing 'REFERENCES' and 'NOT NULL'

### ISSUE THREAD

**Issue by Nerwerder** [xnuinside/simple-ddl-parser#239]
**Describe the bug**

Calling 'parse_from_file' on a file containing (postgres) sql 'create table' instructions will not return tables containing a column using 'REFERENCES' and 'NOT NULL'.

**To Reproduce**

Good case:
```
CREATE TABLE a
(
    id UUID PRIMARY KEY
);

CREATE TABLE b
(
    id UUID PRIMARY KEY,
    a_id UUID REFERENCES a(id)
);
```
=> Will return two tables.

Bad case:
```
CREATE TABLE a
(
    id UUID PRIMARY KEY
);

CREATE TABLE b
(
    id UUID PRIMARY KEY,
    a_id UUID REFERENCES a(id) NOT NULL
);
```
=> Will only return a single table (a).

**Expected behavior**

All tables should be returned.

**Environment:**
 - Ubuntu 23.10
 - Python 3.12.0
 - simple-ddl-parser 1.0.3

**Comment by xnuinside** [comment]
fixed in version 1.0.4 https://github.com/xnuinside/simple-ddl-parser/pull/244

@Nerwerder, thanks for opening the issue!

**Comment by xnuinside** [comment]
version 1.0.4 was released, issue was fixed, if needed anything else - feel free to open a new issue
